packages/client/wallets/web3auth-adapter/src/index.ts

```diff
@@ -17,13 +17,14 @@ import {
     getUrlProviderByBlockchain,
 } from "./networks";
 
+export type { TORUS_NETWORK_TYPE } from "@web3auth/single-factor-auth";
+export { EVMBlockchainIncludingTestnet as Blockchain } from "@crossmint/common-sdk-base";
 export type Web3AuthSignerParams = {
-    type: "WEB3_AUTH";
```

Comment:
wait this isn't redundant, it makes our life easy and IMO better experience for the developer

Reply:
maybe I missed something, previously we've used this as a param for `getOrCreateWallet`, like:
```
const signer = {
  type: "WEB3_AUTH",
  ...
}
getOrCreateWallet(user, signer)
```
do we now use `getWeb3AuthSigner` instead?